src/entries/popup/pages/home/Points/usePoints.ts

```diff
@@ -25,7 +28,7 @@ export const fetchPointsQuery = async (address: Address) =>
 let nextDropTimeout: NodeJS.Timeout | undefined;
 export const usePoints = (address: Address) => {
   const query = useQuery({
-    queryKey: ['points', address],
+    queryKey: ['points', address, PointsDocument.loc?.source.body],
```

Comment:
updating the queryKey when the query changes dismisses the cache and force a refetch, so we don't have to check if `last_airdrop` exists every time

Reply:
oh i see